ZeachCobbler.user.js

```diff
@@ -5065,11 +5092,15 @@ $('#AcidMode-checkbox').change(function(){
     if(!!this.checked){
         $('#AcidMode-textbox').removeAttr("disabled");
     } else {
-        $('#AcidMode-textbox').attr({disabled:"disabled"})
+        $('#AcidMode-textbox').attr({disabled:"disabled"});
     }
     cobbler.MenuAcidMode = !!this.checked;
 });
-if(cobbler.MenuAcidMode){$('#AcidMode-checkbox').prop('checked', true);}else{ $('#AcidMode-textbox').attr({disabled:"disabled"})}
```

Comment:
On 13.12.2015 23:51, Pepin wrote:
>
> In ZeachCobbler.user.js
> <https://github.com/RealDebugMonkey/ZeachCobbler/pull/78#discussion_r47451859>:
>
> >      }
> >      cobbler.MenuAcidMode = !!this.checked;
> >  });
> > -if(cobbler.MenuAcidMode){$('#AcidMode-checkbox').prop('checked', true);}else{ $('#AcidMode-textbox').attr({disabled:"disabled"})}
>
> Q: Why you here deleting the line and "expanding it"
>
I noticed this line (an dall the others) because of the missing
semicolon at the end.
I could have left the rest of the statement alone but it's more readable
on multiple lines, e.g. you can see more easily if one of the CSS tags
is misspelled.
>
> (sorry, my English is very high level)?
>
No problem.

-- 
-- Matthias Urlichs